docs/core/config/sieve/index.md

```diff
@@ -188,22 +189,26 @@ global scripts using the `sievec` command line tool. For example:
 sievec /var/lib/dovecot/sieve/global/
 ```
 
-This is necessary for scripts listed in [[setting,sieve_global]],
-[[setting,sieve_before]], and [[setting,sieve_after]] settings.
+This is necessary for script in storages with `after`, `before`, `global`, and
+`default` storage type (see [[link,sieve_storage_type]]).
 
 For global scripts that are only included in other scripts using the Sieve
-include extension, this step is not necessary since included scripts
-are incorporated into the binary produced for the main script.
+include extension (from the `personal` and `global` storage types), this step is
+not necessary since included scripts are incorporated into the binary produced
+for the main script.
 
 ## Compile and Runtime Logging
 
 Log messages produced during script compilation or during script
 execution are written to two locations by the LDA Sieve plugin:
 
-- A log file is written in the same directory as the user's main
-  private script (as specified by [[setting,sieve]]). This
-  log file bears the name of that script file appended with ".log", e.g.
-  `.dovecot.sieve.log`.
+- If the user's personal storage is using the `file` driver
+  (see [[link,sieve_storage_driver]]), a log   file is written in the same
```

Comment:
`using the [[link,sieve_storage_file,file driver]], a log file ..` (there also extra space now)

Reply:
Changed